programmers/짝수와-홀수.js

```diff
@@ -0,0 +1,12 @@
+function solution(num) {
+    var number
+    if (num %2 ===0){
+        number = "Even";
+    }
+    else{
+        number = "Odd";
+    }
+    return number;
+}
+var num = 4 ;
+console.log(solution(num));
```

Comment:
배열로 풀수도 있습니다.
num % 2가 짝수라면 0 ["Even", "Odd"]의 0번지 Even을 리턴
홀수라면 1 "Odd"를 리턴합니다.
```suggestion
function solution(num) {
  return ["Even", "Odd"][num % 2];
}
```

Reply:
감사합니다! :) 다음에 잘 응용해보겠습니다.